error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/StaticImport.java

```diff
@@ -102,7 +102,8 @@ public final class StaticImport extends BugChecker implements MemberSelectTreeMa
           "org.springframework.http.MediaType",
           "org.testng.Assert",
           "reactor.function.TupleUtils",
-          "tech.picnic.errorprone.bugpatterns.util.MoreTypes");
+          "tech.picnic.errorprone.bugpatterns.util.MoreTypes",
+          "tech.picnic.errorprone.bugpatterns.util.ThirdPartyLibrary");
```

Comment:
This doesn't cause the enum values to be statically imported. Need to investigate before merging this PR.